### ISSUE TITLE
The lock screen redesign looks bad on smaller devices 

### ISSUE THREAD

**Issue by dkasak** [signalapp/Signal-Android#7594]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The redesigned lock screen looks bad on smaller devices since the grey container box (and its padding) make the text box too narrow which makes the text "Enter passphrase" only partially visible. Also, the Signal logo seems to be placed a bit too high since it touches the status bar (better visible on the light theme).

IMO, the redesign looks worse overall than what was there before.

I'm not sure how to properly resolve it, but a potential solution could be:

1. Make the text box wider (reducing the grey container box left and right padding)
2. Place the Signal logo a bit closer to the container box.

### Screenshots
![screenshot_20180401-113614-resized](https://user-images.githubusercontent.com/463342/38172080-efa6eb04-35a5-11e8-9ce7-4d8bb4d53af3.png)

### Device info
**Device:** Samsung Galaxy S4 Mini (GT-I9195)
**Android version:** 7.1.2
**Signal version:** 4.17.5

**Comment by moxie0** [comment]
The signal-based passphrase lock isn't really an ongoing supported feature. You're seeing it because you had it enabled before, but everything has been updated to support fingerprint or system pattern/pin/passphrase unlock. If you toggle it off in settings you'll see the setting to enable the new lock stuff.

**Comment by dkasak** [comment]
Yeah, I've only seen that there is consolidation happening between Signal's unlocking mechanism and the system one after I've already posted the issue. It sucks a bit because I know a lot of Signal users who were relying on the auth secret for Signal being different from the system one.

However, when I enabled the new unified screen lock mechanism, the explicit "Lock" option in the menu disappeared too. Is there now also no way to lock Signal explicitly, prior to the timeout elapsing?
